src/main/resources/swagger/api-docs.yaml

```diff
@@ -618,6 +620,13 @@ definitions:
         description: |
           The date and time the cluster was last accessed, in ISO-8601 format.
           Date accessed is defined as the last time the cluster was created, modified, or accessed via the proxy.
+      autopause:
+        type: boolean
+        description: Whether autopause feature is enabled for this specific cluster.
+      autopauseThreshold:
+        type: int
+        description: The number of minutes to elapse before the cluster is autopaused. If autopause is set to false, this value is disregarded. A value of 0 is equivalent to autopause being turned off.
```

Comment:
Good text - only comment is to make it clear that the cluster only auto-pauses after it's _idle_ for the threshold. Maybe "The number of minutes of idle time to elapse..." ?